import/import.go

```diff
@@ -0,0 +1,534 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+
+	"golang.org/x/exp/maps"
+
+	"github.com/astronomer/terraform-provider-astro/internal/clients/iam"
+
+	"github.com/astronomer/terraform-provider-astro/internal/clients"
+
+	"github.com/astronomer/terraform-provider-astro/internal/clients/platform"
+	"github.com/samber/lo"
+)
+
+type HandlerResult struct {
+	Resource string
+	Error    error
+}
+
+func main() {
+	log.SetFlags(0)
+	log.Println("Terraform Import Script Starting")
+
+	// collect all arguments from the user, indicating all the resources that need to be imported
+	resourcesPtr := flag.String("resources", "", "Comma separated list of resources to import")
```

Comment:
we should have enums and validation on these

Reply:
ie if someone puts in something incorrect like `wokrspace`, we should fail and tell them `wokrspace` is invalid and these are the valid ones